intern/service/boss/baseversion.py

```diff
@@ -147,7 +147,7 @@ def build_metadata_url(self, resource, url_prefix, key, value=None):
         return urlWithKey + '&value=' + str(value)
 
     def build_cutout_url(
-        self, resource, url_prefix, resolution, x_range, y_range, z_range,  time_range=None, id_list=[], no_cache=False):
+        self, resource, url_prefix, resolution, x_range, y_range, z_range,  access_mode, time_range=None, id_list=[]):
```

Comment:
will reordering arguments cause an issue? even with optional arguments, if they are all specified without the keyword this will cause a problem.